daemon/cmd/endpoint.go

```diff
@@ -453,6 +454,15 @@ func (d *Daemon) createEndpoint(ctx context.Context, owner regeneration.Owner, e
 				}).Warningf("Endpoint has %s annotation, but BPF bandwidth manager is disabled. This annotation is ignored.",
 					bandwidth.EgressBandwidth)
 			}
+			if hwAddr, ok := annotations["cni.cilium.io/hwAddr"]; ok {
```

Comment:
One more minor nit, should this be named something like `cni.cilium.io/macAddr`? I think `hw` expands to `hardware` which is not strictly true given we're talking about virtual device configuration.

Reply:
I borrowed from [Calico](https://docs.tigera.io/calico/latest/networking/configuring/pod-mac-address), not sure if this is the best practice.

Reply:
The label is already different, I'm not sure if there is much point in retaining a partial name match :shrug: